packages/grafana-toolkit/src/config/webpack.plugin.config.ts

```diff
@@ -151,11 +153,7 @@ export const getWebpackConfig: WebpackConfigurationGetter = options => {
       '@grafana/data',
       // @ts-ignore
       (context, request, callback) => {
-        let prefix = 'app/';
-        if (request.indexOf(prefix) === 0) {
-          return callback(null, request);
-        }
-        prefix = 'grafana/';
+        const prefix = 'grafana/';
```

Comment:
Why this change? Related to https://github.com/grafana/grafana/pull/17850#issuecomment-507635571 ?

Reply:
more related to imports from 'app/..' directly do not really work without some big hoops and using references:
https://github.com/ryantxu/ajax-panel/blob/master/src/module.ts#L1

I took it out and will see if I can get all the plugins migrated to toolkit without it.  If not, we can come back and support it

Reply:
I don't think we will be able to build any of the plugins without the reference comments. Here we are only saying to webpack to externalise `[app|grafana]/` imports. It's the TS that's gonna fail on type declarations for those modules.